lib/factory.ml

```diff
@@ -160,7 +160,7 @@ module Sig = struct
     match ptype_kind with
     | Ptype_record labels -> from_record ~loc ~type_name ~return_type ~labels
     | Ptype_variant ctors -> List.map (from_constructor ~loc ~type_name ~return_type) ctors
-    | Ptype_abstract -> Raise.Factory.unhandled_type_kind ~loc "abstract"
+    | Ptype_abstract -> []
```

Comment:
Its weird that this doesn't break any tests? Should we add a test for this behaviour?

Reply:
That's because we don't test errors atm. I'd like to be able to test errors and their locations but I need to figure out an easy way to do that first!